Ekaterina_Musatova_дз3/Ekaterina_Musatova_дз3/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Ekaterina_Musatova_дз3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double y, x, x2;
+            Console.WriteLine("enter x:");
+            x = double.Parse(Console.ReadLine());
+            x2 = Math.Pow(x, 2);
```

Comment:
Лучше использовать умножение
```suggestion
            x2 = x * x;
```